tools/cldr-apps/src/org/unicode/cldr/util/CLDRConfigImpl.java

```diff
@@ -180,7 +171,7 @@ private synchronized void init() {
         // The git version of the cldr.jar file embedded in cldr-apps.
         survprops.put("CLDR_UTILITIES_HASH", getGitHashForSlug("CLDR-Tools"));
         // The git version of the CLDR_DIR currently in use.
-        survprops.put("CLDR_DATA_HASH", getGitHashForDir(survprops.getProperty("CLDR_DIR", null)));
+        survprops.put("CLDR_DATA_HASH", CldrUtility.getGitHashForDir(survprops.getProperty("CLDR_DIR", null)));
```

Comment:
There's no comnon init() for CLDRConfig, but if there were, this line could move there.

Reply:
At first I tried to call getCldrBaseDirHash there but got "Error: setDir() was already called once." Somehow putting the cart before the horse, I guess -- getCldrBaseDirHash calls getCldrBaseDirectory...